README.md

```diff
@@ -1,48 +1,78 @@
 ```
 
-Next, from the root of u8timeseries:
+##### Running the examples only without installing:
+To run the example notebooks without installing, using Docker, you can also run: 
 ```
-pip install .
+./build_docker.sh && ./run_docker.sh
 ```
+Then copy and paste the URL provided by the docker container into your browser to access Jupyter notebook.
 
-If Fortran is not installed on the device, the following error message might be printed during the installation:
+### Example Usage
+Create `TimeSeries` object from a Pandas DataFrame, and split in train/validation series:
+```python
+from u8timeseries import TimeSeries
 
-```
-error: library dfftpack has Fortran sources but no Fortran compiler found
-```
-This can be solved by installing gcc using Homebrew:
-```
-brew install gcc
+df = pd.read_csv('AirPassengers.csv', delimiter=",")
+series = TimeSeries.from_dataframe(df, 'Month', '#Passengers')
+train, val = series.split_after(pd.Timestamp('19590101'))
 ```
 
-## docker install
+Fit an exponential smoothing model, and make a prediction over the validation series' duration:
+```python
+from u8timeseries import ExponentialSmoothing
 
-Build and run the docker using the following two commands:
+model = ExponentialSmoothing()
+model.fit(train)
+prediction = model.predict(len(val))
 ```
-./build_docker.sh
-./run_docker.sh
+
+Plot:
+```python
+series.plot(label='actual', lw=3)
+prediction.plot(label='forecast', lw=3)
+plt.legend()
+plt.xlabel('Year')
 ```
-Then copy and paste the URL provided by the docker container into your browser to access jupyter notebook.
+![example](images/example.png "example")
 
-## Usage
-For now the best documentation is examples.
-See: https://github.com/unit8co/u8timeseries/blob/master/examples/Air-passengers-example.ipynb
+We invite you to go over the example notebooks in the `examples` directory.
 
-## Issue with Prophet and Pandas
-If you encounter the following error when trying to plot a `TimeSeries`:
-```
-float() argument must be a string or a number, not 'Period
-```
-this is likely because [Prophet deregisters the Pandas converters in its code](https://darektidwell.com/typeerror-float-argument-must-be-a-string-or-a-number-not-period-facebook-prophet-and-pandas/). To fix it, just call
-```
-pd.plotting.register_matplotlib_converters()
-```
+### Documentation
+The documentation of the API and models is available here:
+https://unit8co.github.io/u8timeseries/
+
+### List of Features
+Currently, the library contains the following features: 
+
+* **Forecasting Models:** Exponential smoothing, ARIMA & auto-ARIMA, Facebook Prophet, Theta method, 
+Recurrent neural networks (vanilla RNNs, GRU, and LSTM variants).
+
+* **Preprocessing:** Transformer tool for easily scaling / normalizing time series.
+
+* **Metrics:** A variety of metrics for evaluating time series' goodness of fit; 
+from R2-scores to Mean Absolute Scaled Error.
+
+* **Backtesting:** Utilities for simulating historical forecasts, using moving time windows.
```

Comment:
I might be wrong, but it seems to me that the historical forecast currently utilises an expanding window approach rather than a moving window approach at u8timeseries/u8timeseries/backtesting/forecasting_simulation.py at line 75.